src/cql/upstream-external/CirgLibraryQuestionnaireLogic_DCW.cql

```diff
@@ -384,14 +394,15 @@ define PresentQnr_CIRG_GDS:
   } else null
 
 define PresentQnr_CIRG_C_IDAS:
+  // check if there are already questionnaire responses for today
   if Length(CIRG_C_IDAS_Responses_Today) >= 1 then null
-  // CIRG-SLUMS score indicates dementia 
-  else if dcw_mood_level_2 is true and 
-     dcw_care_partner_present is true and 
-     HasSLUMSDementiaScore is true
+  // dementia diagnosis/condition or CIRG-SLUMS score indicates dementia 
+  else if HasDementia or (
+    dcw_mood_level_2 is true and 
+    dcw_care_partner_present is true and 
+    HasSLUMSDementiaScore is true)
```

Comment:
I believe this is incorrect and that the conditions should be:
CP present AND Dementia from SLUMS.
Per [our curated gdoc](https://docs.google.com/document/d/1_6tiOwgPt8xMvLrRR8U234X-31ZhYlK2/edit#bookmark=id.1zczfw5557ti) and confirmed upstream in [sharepoint](https://uwnetid.sharepoint.com/:w:/r/sites/og_son_dpen_team/_layouts/15/Doc.aspx?sourcedoc=%7BD118B699-1D5E-4C0A-8931-422CCE5E7306%7D&file=DCW_8.9.22.docx&action=default&mobileredirect=true).

Looks like the tests will need editing too.

Reply:
@mcjustin hmm, okay, this change is in response to the story:  https://www.pivotaltracker.com/story/show/184364472  It sounds like a Dementia diagnosis/condition should also trigger administration of SLUMS?

Reply:
this looks correct to me! it is SLUMs indicating dementia OR dementia DX

Reply:
@sierramatice  I see that you edited the spec - thanks. Copying it here for convenience:
```
Condition: CP gateway question: CP present 
AND
Condition: Dementia from SLUMS OR dementia DX
```
And noting the overall mood/behavior level 2 criteria of:
```
depression or anxiety diagnosed in the EMR 
Behave5+ failed 
```
The proposed change here, however, does not match what you wrote. 
```
  else if HasDementia or (
    dcw_mood_level_2 is true and 
    dcw_care_partner_present is true and 
    HasSLUMSDementiaScore is true)
```
I'm on calls back to back today, but that's my 2 cents...

Reply:
Oh! I misinterpreted the code, I focused on the comment `// dementia diagnosis/condition or CIRG-SLUMS score indicates dementia `

@achen2401 the care_partner_present always needs to be true to present this assessment!